core/templates/services/literally-canvas-helper.service.ts

```diff
@@ -0,0 +1,327 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview LiterallyCanvas editor helper service.
+ */
+
+// TODO(#9186): Change variable name to 'constants' once this file
+// is migrated to Angular.
+const CONSTANTS = require('constants.ts');
+
+angular.module('oppia').factory('LiterallyCanvasHelperService', [
+  function() {
+    const getPoints = (x, y, angle, width, length) => [
+      {
+        x: x + (Math.cos(angle + Math.PI / 2) * width) / 2,
+        y: y + (Math.sin(angle + Math.PI / 2) * width) / 2,
+      },
+      {
+        x: x + Math.cos(angle) * length,
+        y: y + Math.sin(angle) * length,
+      },
+      {
+        x: x + (Math.cos(angle - Math.PI / 2) * width) / 2,
+        y: y + (Math.sin(angle - Math.PI / 2) * width) / 2,
+      },
+    ];
+    const arrow = {
```

Comment:
Please add single newlines between large blocks of code (e.g. here and above the return statement below). Otherwise it becomes hard to read.

Reply:
Done.